crates/notedeck_columns/src/app.rs

```diff
@@ -568,6 +569,8 @@ fn render_damus_mobile(
                 }
             }
         }
+    } else {
+        app.columns_mut(app_ctx.accounts).new_column_picker();
```

Comment:
we should probably do this before the call to render_nav

Reply:
@jb55 my bad, you're right... I've moved it to `update_damus`